examples/basic/packages/ui/package.json

```diff
@@ -1,21 +1,36 @@
 {
-  "name": "ui",
+  "name": "@repo/ui",
   "version": "0.0.0",
-  "main": "./index.tsx",
-  "types": "./index.tsx",
+  "private": true,
   "license": "MIT",
+  "exports": {
+    "./card": {
+      "types": "./src/card.tsx",
+      "development": "./src/card.tsx",
+      "default": "./dist/card.js"
+    },
+    "./code": {
+      "types": "./src/code.tsx",
+      "development": "./src/code.tsx",
+      "default": "./dist/code.js"
+    }
+  },
```

Comment:
Can't we just do:

```
  "exports": {
    "./*": {
      "types": "./src/*.tsx",
      "development": "./src/*.tsx",
      "default": "./dist/*.js"
    }
  },
```

Reply:
That's this. https://github.com/vercel/turbo/pull/6439#issuecomment-1813839498

TL;DR: If you wildcard export, you don't get auto-imports across workspaces.

Reply:
Ah. such sad